src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Entities/ValidationConstants.cs

```diff
@@ -13,14 +13,16 @@ internal static class ValidationConstants
         /// Tournament entity validation constants
         /// </summary>
         public static class Tournament
-        {
+        {       
             public const short SCHEMA_STORAGE_OFFSET = 1900;
 
             public const int MAX_NAME_LENGTH = 60;
 
             public const int MAX_DESCRIPTION_LENGTH = 300;
 
             public const int MAX_URL_LENGTH = 255;
+
+            public static int MAX_LOCATION_LENGTH = 50;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'MAX_LOCATION_LENGTH' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2223)